src/_pytest/main.py

```diff
@@ -228,6 +228,23 @@ def pytest_addoption(parser: Parser) -> None:
         ),
     )
 
+    group.addoption(
```

Comment:
Let's make those configuration variables instead, seems like something users will configure once instead of passing them in the command-line all the time.

Reply:
Done. 
I was confused with command line options and configuration options.